src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -37,12 +39,29 @@ object Run  {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  private def dateTryToString(dateTry: Try[DateTime]): String = dateTry match {
+    case scala.util.Success(d: DateTime) => d.toString
+    case Failure(e: Throwable) => e.getClass.getName
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  final case class Initializing(created: Try[DateTime]) extends RunStatus {
+    override def toString = s"Initializing(created: ${dateTryToString(created)})"
+  }
+
+  final case class Running(created: Try[DateTime], started: Try[DateTime]) extends RunStatus {
```

Comment:
Thinking about this a bit further I think the right thing to do here is to just not bother capturing the timestamps in the non-terminal states. When I thought back to why I put it in there in the first place it was because I was trying to figure out WTH things were taking so long w/o knowing yet that it was just due to that JES delay. 

The reality is that from the perspective of Cromwell a JES job is launched and then is a black box until it is completed. So perhaps instead of logging all of this stuff we should have something like "Sending task foo to JES" and then a message when it hits a terminal state w/ the timestamps.

As for how to deal with missing timestamps on a terminal state, I'm still thinking that that seems like a bad thing but it seems weird to fail the whole workflow. Perhaps something like try another time or two before blowing up?